instance/models/database_server.py

```diff
@@ -120,6 +121,10 @@ class DatabaseServer(ValidateModelMixin, TimeStampedModel):
     class Meta:
         abstract = True
 
+    # Human readable identifier for database instances
+    name = models.CharField(max_length=128, unique=True)
+    description = models.CharField(max_length=128, blank=True)
```

Comment:
Keep in mind that we are approaching some pylint limits on how many mixin classes we have, and it can get hard to follow the code paths when there are too many mixins. So a more thoughtful refactor may be required.